WebApp/src/uk/ac/exeter/QuinCe/web/datasets/ExportBean.java

```diff
@@ -327,15 +329,18 @@ private void exportDatasetWithRawFiles() {
           output.append(exportOption.getSeparator());
         }
 
-        for (String calculatedColumn : exportOption.getCalculationColumns("equilibrator_pco2")) {
-          Double value = calculationRecord.getNumericValue(calculatedColumn);
-          if (null == value) {
-            output.append("NaN");
-          } else {
-            output.append(numberFormatter.format(value));
+        List<String> calculationColumns = exportOption.getCalculationColumns("equilibrator_pco2");
+        if (null != calculationColumns) {
+          for (String calculatedColumn : calculationColumns) {
+            Double value = calculationRecord.getNumericValue(calculatedColumn);
+            if (null == value) {
+              output.append("NaN");
```

Comment:
Is the NaN here the only option for all exports? Should this at least be a variable / constant?

Reply:
Good question. It should probably be an option in the export config, since what we use will depend on the needs to the final destination. I've created an issue (#1062) for a future release, and if anyone brings it up we can bring it forward.